colour/models/rgb/transfer_functions/__init__.py

```diff
@@ -170,7 +173,7 @@ def log_encoding(value, function='Cineon', **kwargs):
         'Canon Log 3', 'Canon Log', 'Cineon', 'D-Log', 'ERIMM RGB', 'F-Log',
         'Filmic Pro 6', 'Log3G10', 'Log3G12', 'Panalog', 'PLog', 'Protune',
         'REDLog', 'REDLogFilm', 'S-Log', 'S-Log2', 'S-Log3', 'T-Log',
-        'V-Log', 'ViperLog'}**,
+        'V-Log', 'ViperLog', 'Log2'}**,
```

Comment:
And same for the remaining instances (even in the docs).

Reply:
Will take care, Thankyou!